frontend/src/components/roadmap-survey/roadmap-survey.test.tsx

```diff
@@ -0,0 +1,45 @@
+import { Group } from '@api/models/group';
+import { render, screen } from 'test-utils';
+import mockManyGroups from 'utils/mocks/groups';
+import RoadmapSurvey from './roadmap-survey';
+
+let groups: Group[] = [];
+
+describe('RoadmapSurvey', () => {
+  beforeEach(() => {
+    groups = JSON.parse(JSON.stringify(mockManyGroups));
+  });
+
+  it('should render component with all groups', () => {
+    const rendered = render(<RoadmapSurvey groups={groups} />);
+    groups.map((group) =>
+      expect(rendered.getByTestId(`group-${group.id}`)).toBeInTheDocument()
+    );
```

Comment:
In these cases it is better to use a `forEach` instead of a `map`, as we are not returning anything.

Reply:
Nice!

Reply:
@iurynogueira  you resolved this conversation and I merged the PR. I just checked again after emerging and there are some other places where maps is being used but it could be replaced by each. I will create an issue to keep that refactor in mind